test/source/mock/google/google-data.ts

```diff
@@ -186,7 +186,8 @@ export class GoogleData {
     }
     let body: GmailMsg$payload$body;
     if (parsedMail.text) {
-      body = { data: parsedMail.text, size: parsedMail.text.length };
+      const data = parsedMail.html || parsedMail.text;
```

Comment:
A cleaner way to write it would be to add `if(parsedMail.html)` as the first condition above

Reply:
done 👍